tests/utils/warp-genesis.json

```diff
@@ -22,7 +22,7 @@
       "blockGasCostStep": 500000
     },
     "warpConfig": {
-      "blockTimestamp": 0
+      "blockTimestamp": 1719343601
```

Comment:
why do we need to specify a specific warp blocktimestamp, as well as the timestamp value below?

Reply:
I was experiencing a hang in the test setup, and @ceyonur said "warp cannot be activated at timestamp = 0...also set genesis timestamp  to some non-zero value (a recent timestamp is fine)...it was changed after v0.6.3 when we introduced modifiable network upgrades and fixing a regression with using 0 timestamps in genesis." starting from https://platwizards.slack.com/archives/C075HLQL4BT/p1719327426806459?thread_ts=1718898895.417379&cid=C075HLQL4BT

it looks like this is the original issue as reported: https://github.com/ava-labs/avalanchego/issues/2877, which was fixed by https://github.com/ava-labs/subnet-evm/pull/1136 and included in https://github.com/ava-labs/subnet-evm/releases/tag/v0.6.4